### PR TITLE
SSVD-2462 Potential fix for color temperature debouncing using runIns

### PR DESCRIPTION
**Pull request by workingmonk**
cc @tpmanley

### CODE REVIEW

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 1]
```diff
@@ -85,6 +85,9 @@ def parse(String description) {
     if (finalResult) {
         log.debug finalResult
         if (finalResult.name=="level" && finalResult.value==0) {}
+        else if (event.name=="colorTemperature") {
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: I think this should be finalResult.name

Comment (workingmonk) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: good catch

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 2]
```diff
@@ -85,7 +85,7 @@ def parse(String description) {
     if (finalResult) {
         log.debug finalResult
         if (finalResult.name=="level" && finalResult.value==0) {}
-        else if (event.name=="colorTemperature") {
+        else if (finalResult.name=="colorTemperature") {
             runIn(5, "sendColorTemperatureEvent", [data: [value: event.value], overwrite: true])
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: Here too

**Comment by workingmonk** [comment]
@tpmanley fixed all the issues

**Comment by tpmanley** [comment]
:+1:

**Pull request closed without merging**